dev/release-aux/openssl-announce-release.tmpl

```diff
@@ -11,10 +11,10 @@
 
         https://www.openssl.org/news/openssl-$series-notes.html
 
-   Specific notes on upgrading to OpenSSL $series from previous versions, as well
-   as known issues are available on the OpenSSL Wiki, here:
+   Specific notes on upgrading to OpenSSL $series from previous versions are
+   available in the OpenSSL Migration Guide, here:
 
-        https://wiki.openssl.org/index.php/OpenSSL_$series
+        https://www.openssl.org/docs/man$series/man7/migration_guide.html
 
```

Comment:
Have you tested this expands correctly?

I don't know how to verify that, otherwise everything LGTM

Reply:
You have to run the release script. It expands as follows (i.e. as expected):

````
   Specific notes on upgrading to OpenSSL 3.0 from previous versions are
   available in the OpenSSL Migration Guide, here:

        https://www.openssl.org/docs/man3.0/man7/migration_guide.html
````

Reply:
Which doesn't currently exist ... which I think is what @roman is pointing out

Reply:
@romen given the discussion above does the approval still stand?

Reply:
It does!